style.css

```diff
@@ -327,6 +327,126 @@ button:hover {
   margin-top: 2rem;
 }
 
+/* ---Popup Window-- */
+
+.popup {
+  visibility: visible;
+  position: fixed;
+  z-index: 60;
+  height: 813px;
+  padding-top: 0.5rem;
+  width: 88%;
+  left: 4.5%;
+  top: 7.6rem;
+  border-radius: 16px;
+  margin-top: -7rem;
+  align-items: center;
+  background-color: white;
+}
+
```

Comment:
- You have done great so far :+1: : generating the project section dynamically :+1: :+1: But there are som changes you still need to make before moving to the next milestone. :+1: 
- [ ] Looking at the screenshot below, you can see that the desktop pop-up doesn't look like the Figma design when on the screen size of `1450 px` and above, the language list and the button tends to flow onto the Image. I'll suggest you fix this. :+1: :+1: 

- - Screenshot:
![Selection_005](https://github.com/denigit5/Portfolio/assets/107632780/7bba4da2-9c8a-46bf-a4c4-4d89d794e395)

- [ ] Also looking at the screenshot below, you can see that when on screen sizes below `1340 px` the image on the pop-up doesn't resize and therefore the full image is not visible, part of the image is cropped out. I'll suggest you fix this also. :+1: :+1: 

- - Screenshot:
 
![Selection_006](https://github.com/denigit5/Portfolio/assets/107632780/1d715e04-af98-4bfb-87e9-274a71d2c53b)